server/terraform/modules/myhealth/main.tf

```diff
@@ -257,3 +259,62 @@ resource "google_compute_backend_service" "backend" {
   health_checks = null
   depends_on    = [google_project.project]
 }
+
+
+# Google Cloud bucket for static content assets.
+resource "google_storage_bucket" "content" {
+  name     = "${var.project_id}-static-content" #################### Should a prefix be added to make uniqueness more likley? 
+  location = var.region
+
+  # There should be no need for per-object ACLs with this bucket.
+  uniform_bucket_level_access = true
+
+  ############################################################
+  # Do we need to set the lifecycle rule or object versioning? I expect not, or with a single version / short lifespan?
```

Comment:
We probably don't need to add `prevent_destroy` but it may prevent an accident or two.

Reply:
Ah, I was actually talking about the policy of the bucket, not the terraform management.

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#lifecycle_rule

I think it's OK to leave this the default, as the contents of this bucket should all be pushed from an external source, the bucket is a distribution mechanism not the source of the data.